src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/ValidationError.cs

```diff
@@ -161,8 +161,8 @@ internal Exception CreateException(Type exceptionType, Exception? innerException
                     exception = new SecurityTokenInvalidIssuerException(MessageDetail.Message, innerException);
                 else if (exceptionType == typeof(SecurityTokenSignatureKeyNotFoundException))
                     exception = new SecurityTokenSignatureKeyNotFoundException(MessageDetail.Message, innerException);
-                else if (exceptionType == typeof(SecurityTokenDecryptionFailedException))
```

Comment:
Why remove this piece of code?

Reply:
Trying to keep the changes in the PR minimal to the issue at hand.